internal/x11/win/frame.go

```diff
@@ -35,6 +37,10 @@ type frame struct {
 	borderTop, borderTopRight xproto.Pixmap
 	borderTopWidth            uint16
 
+	doubleClick bool
+	ctx context.Context
```

Comment:
This variable (and possibly others?) can be limited to the function that fires off the goroutine can it not?

p.s. `doubleClick` sounds like a verb, perhaps something like 'clickCount' or other name might better indicate it's purpose?